lib/cucumber/rails/configuration.rb

```diff
@@ -0,0 +1,26 @@
+module Cucumber
+  module Rails
+    class Configuration
+      attr_writer :use_rack_test_helpers
+
+      def use_rack_test_helpers
+        return @use_rack_test_helpers if defined? @use_rack_test_helpers
```

Comment:
You can simplify this method by removing these two lines and replacing them with `@use_rack_test_helpers || true`. If `@use_rack_test_helpers` is undefined it will be evaluated to false, thus `true` would be returned. :)

Reply:
Not sure this will work: if `@use_rack_test_helpers` is defined, but equals `false` then the result of this line will be `true`, i.e. the default value. This is not what we want. Am I missing something?

Reply:
Oops! Good point.